src/Components/InfoContainer/LogTypeMapContainer/LogTypeMapContainer.js

```
@@ -0,0 +1,29 @@
+import React, {useEffect, useRef} from "react";
+
+import "./LogTypeMapContainer.scss";
+
+/**
+ * @return {JSX.Element}
+ */
+export function LogTypeMapContainer ({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)